source/manual/deployment-dashboards.html.md

```diff
@@ -4,7 +4,7 @@ title: Monitor your app during deployment
 section: Deployment
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2017-11-14
+last_reviewed_on: 2018-05-21
```

Comment:
Is this article valuable beyond the first 4 paragraphs? The rest seems self-explanatory - could we lose it?